docs/en/solutions/apply_catboost_model.md

```diff
@@ -0,0 +1,204 @@
+# Applying a CatBoost model in ClickHouse {#applying-catboost-model-in-clickhouse}
+
+[CatBoost](https://catboost.ai) — is a free and open-source gradient boosting library for machine learning.
+
+To apply a CatBoost model in ClickHouse:
+
+1. [Create a table](#create-table).
+2. [Insert the data to the table](#insert-the-data-to-the-table).
+3. [Configure the model](#configure-the-model).
+4. [Run the model inference from SQL](#run-the-model-inference).
+
+For more information about training CatBoost models, see [Training and applying models](https://catboost.ai/docs/features/training.html#training).
+
+## Before you start {#before-you-start}
+
+If you don't have the [Docker](https://docs.docker.com/install/) yet, install it.
+
+> **Note:** [Docker](https://www.docker.com) uses containers to create virtual environments that isolate a CatBoost and ClickHouse installation from the rest of the system. CatBoost and ClickHouse programs are run within this virtual environment.
```

Comment:
* incorrect note block styling
* "virtual environments" is often used to name different kind of isolation, these are called containers
* CatBoost has a console tool, but it's not used in this tutorial, so last sentence is kind of out of place